build-system/server/new-server/transforms/utilities/cdn.ts

```diff
@@ -32,11 +34,12 @@ export function CDNURLToLocalDistURL(
   url.port = '8000';
 
   const [overwriteablePathname, newPathname] = pathnames;
+  const outputDir = argv.sxg ? 'sxg' : 'dist'
```

Comment:
Wondering if you should use a helper function here for outputDir, since this is duplicated from the other file. Seems like an opportunity to forget to update if it ever changes?

Reply:
Ah, I see there is a helper. Just use it here and in bundles.config.js ?